src/react/features/rooms/effects/useGetHomeEffect.js

```diff
@@ -7,12 +7,22 @@ import { setConnections } from '../../connectors/actions'
 
 import { invokeChannel } from '../../../client/client'
 import { channels } from '../../../../shared/constants'
-
+import { invokeAlertDialog } from '../../../client/alertDialog'
+/*
+ * Requests home & focus info via GET_HOME and sets the focus for room & diory
+ * - saves the response to the store
+ * - invoke an alert for each error
+ */
 export const useGetHomeEffect = () => {
   const dispatch = useDispatch()
 
   useEffect(() => {
-    invokeChannel(channels.GET_HOME).then(({ rooms, connections, focus }) => {
+    invokeChannel(channels.GET_HOME).then(({ rooms, connections, focus, errors }) => {
```

Comment:
invokeChannel(channels.GET_HOME) is something that might require a specific method with a good name for clarity here. getHomeDataFromBackendStore?

Reply:
should we have a method for all invokeChannel(channels...)s?

Reply:
That's something we can think about.
Once we have good test coverage and figured out good names , adding that kind of layer should be a no-brainer.